spec/unit/puppet-strings/yard/parsers/puppet/parser_spec.rb

```diff
@@ -140,6 +140,42 @@ class bar {
     end
   end
 
+  describe 'parsing puppet plans', if: TEST_PUPPET_PLANS do
+    # The parsing code actually checks this and sets a global (Puppet[:tasks]).
+    # Plan parsing will fail if it hasn't yet encountered a file under plans/.
+    let(:file) { 'plans/test.pp' }
+    let(:source) { <<~'SOURCE' }
```

Comment:
I’m using a slightly different style with this heredoc. It seems close to the recommended Ruby style (e.g. `foo(<<EOF)`), though the Ruby style guide doesn’t say anything about single line blocks.

I’m happy to switch over to wrapping it with the block more like above if that’s what’s preferred. Something like:

```ruby
let(:source) do
  <<~'SOURCE'
    # . . .
  SOURCE
end
```

I figured the squiggly heredoc is better regardless (it’s been supported since Ruby 2.3). I can update all the other  heredocs in the tests in a separate PR if you’re interested — I didn’t want to pollute this one.

Reply:
Sounds sensible. I think the work would be a good candidate for a fresh PR 👍